### DISCUSSION TITLE
删除任意词库中任意词组

### DISCUSSION THREAD

**Discussion by blackhole889** [rime/home#739] [Ideas]
rime是一个很好的输入法程序，但也存在一些较大的不足。其中一个就是词库的建立和精选。
提高词库的效率有两个两个方法，我需要的在里面，我不需要的不在里面。只关注其一，如加大词库数量无法提高词库的效率。
现在rime似乎无法删除一些已有词库里的词（其宣称的ctr+del，shift+del,ctr+k可以删除自造词，无法删除一些词库里的词，甚止降低已有词库权重也难以做到。降低已有词库权重偶而可以做到，很不稳定）。有些词一般用户用不到，如果从词库删除可以加大输入效率。能否对原有程序进行修改，或如何配置程序使得可以删除任意词库中任意词组。

**Comment by LEOYoon-Tsaw** [comment]
去xxx.dict.yaml即刻刪除任意詞

**Reply by xtay573269555** [reply]
我用 weasel-0.14.3 版本，使用 ctr+del，shift+del 完全没法删除任何词条，包括自造词条和词库自带的词条。

删除词条这个功能还是很有必要的。

**Comment by redleafnew** [comment]
现在是不是也没有解决这个问题。

**Comment by yanhuacuo** [comment]
只要 schema 结构符合要求，可以使用「rimetool.exe」

![2024-01-21_100139](https://github.com/rime/home/assets/3107994/8c842540-b099-495c-8c44-2739ba8ee9e8)

## 视频演示

https://www.bilibili.com/video/BV1yX4y187Ne/?spm_id_from=333.999.0.0&vd_source=0d05a37563bb1109bc73f16a8004a282

**Reply by redleafnew** [reply]
谢谢。这个还需要用专门工具，如果可以直接在线删词和在线加词就更方便了。

**Reply by yanhuacuo** [reply]
rime 毕竟是跨平台的，输入法前端不可能与后端绑太死，RIME 并不是你常见的或传统认知中的输入法软件。

![2024-01-21_103219](https://github.com/rime/home/assets/3107994/08158abd-e628-4666-bdfd-1f354c112559)

**Reply by redleafnew** [reply]
这个在线加词是什么软件？

**Reply by yanhuacuo** [reply]
rimetool.exe

**Reply by yanhuacuo** [reply]
中州韵98五笔助手，一个跨平台的 RIME 可视化词库、输入法参数管理工具。

**Reply by redleafnew** [reply]
好的，多谢，多谢。

**Reply by mathcae** [reply]
我发现这个工具可以解决 加词  删词 改词
https://github.com/MapoMagpie/rimedm
